docs/basic-usage/teams-permissions.md

```diff
@@ -34,12 +34,12 @@ class TeamsPermission{
     public function handle($request, \Closure $next){
         if(!empty(auth()->user())){
             // session value set on login
-            app(\Spatie\Permission\PermissionRegistrar::class)->setPermissionsTeamId(session('team_id'));
+            setPermissionsTeamId(session('team_id'));
```

Comment:
Is this correct? Is there such a global function?

Reply:
Yes, there is
https://github.com/spatie/laravel-permission/blob/08cc9bdbf89640cb1d64fb01eb9513c1cbb2549d/src/helpers.php#L27-L30